.phpsa.yml

```diff
@@ -95,6 +99,9 @@ phpsa:
         # Checks for use of deprecated functions and gives alternatives if available.
         deprecated_functions:
             enabled:              true
+            check_5_3:            true
+            check_5_5:            true
+            check_7_1:            true
```

Comment:
I suggest to use language_level as @K-Phoen said, because using 100500 configuration is a stupid way